admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -0,0 +1,193 @@
+
+After installing APCu you must restart your Web server. This example restarts 
+Apache on Red Hat/CentOS/Fedora systems::
+
+ systemctl restart httpd.service
+ 
+This example restarts Apache on Debian/Ubuntu/Mint systems::
+
+ service apache2 restart
+ 
+After restarting your Web server, add this line to your ``config.php`` file::
+
+ 'memcache.local' => '\OC\Memcache\APCu',
+ 
+Refresh your ownCloud admin page, and the cache warning should disappear.
+
+Memcached
+---------
+
+Memcached is a reliable oldtimer for shared caching on distributed servers, 
+and performs well with ownCloud with one exception: it is not suitable to use 
+with :doc:`Transactional File Locking <../configuration_files/files_locking_transactional>` because it does not 
+store locks, and data can disappear from the cache at any time (Redis is 
+the best for this). 
+
+Setting up Memcached is easy. This example shows how to install it on 
+Debian/Ubuntu/Mint::
+
+ apt-get install memcached php5-memcached
+
+This example shows how to install and launch it on Red Hat/CentOS/Fedora::
+
+ yum install memcached php-pecl-memcached
+ systemctl enable memcached
+ systemctl start memcached
+ 
+You can verify that the Memcached daemon is running with ``ps``::
+
+ ps ax |grep memcached
+ 19563 ? Sl 0:02 /usr/bin/memcached -m 64 -p 11211 -u memcache -l 
+ 127.0.0.1
+
+Then add the appropriate entries to your ``config.php``. This example uses APCu 
+for the local cache, Memcached as the distributed memcache, and all the 
+servers in the shared cache pool with their port numbers::
+
+ 'memcache.local' => '\OC\Memcache\Memcached',
+ 'memcache.distributed' => '\OC\Memcache\Memcached',
+ 'memcached_servers' => array(
+      array('localhost', 11211),
+      array('server1.example.com', 11211),
+      array('server2.example.com', 11211), 
+      ), 
+
+Redis
+-----
+
+Redis is an excellent modern memcache to use for both distributed caching, and 
+with :doc:`Transactional File Locking <../configuration_files/files_locking_transactional>` because it guarantees 
+that cached objects are available for as long as they are needed. 
+
+This example shows how to install it on Debian/Ubuntu/Mint::
+
+ apt-get install redis-server php5-redis
+
+This example shows how to install and launch it on Red Hat/CentOS/Fedora::
+
+ yum install redis php-pecl-redis
+ systemctl enable redis
+ systemctl start redis
+ 
+You can verify that the redis daemon is running with ``ps``::
+
+ 22203 ? Ssl    0:00 /usr/bin/redis-server 127.0.0.1:6379 
+ 
+This example ``config.php`` configuration uses APCu for the local server cache, 
+and sets up three distributed servers:: 
+
+ 'memcache.local' => '\OC\Memcache\APCu',
+ 'memcache.distributed' => '\OC\Memcache\Redis',     
+ 'redis' => array(
+	'host' => 'localhost', 
+        // optionally, use a Unix socket
+        //'host' => '/tmp/redis.sock',
+	'port' => 6379,
+	'timeout' => 0.0,
+         ),      
+      
+ 'memcached_servers' => array(
```

Comment:
Is this true here? I always thought this can be only used for ``'memcache.distributed' => '\OC\Memcache\Memcached'``